demisto_sdk/commands/generate_docs/README.md

```diff
@@ -1,11 +1,12 @@
-## generate-docs
+# `generate-docs`
 
-Generate a README file for an Integration, Script or a Playbook.
+Generate a `README.md` file for an Integration, Script or a Playbook.
+
+## Use Cases
 
-**Use-Cases**
 This command is used to create a documentation file for Cortex XSOAR content files.
 
-**Arguments**:
+## Arguments
```

Comment:
Are all the `**Example**` to `##Example` changes fine from a BC point of view?
These are the changes that enables support for code like: `{HEADER_TYPE.H3}`, correct?